pkgs/applications/video/gpu-screen-recorder/gpu-screen-recorder-gtk.nix

```
@@ -46,12 +46,12 @@ stdenv.mkDerivation {
     gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libglvnd ]}:/run/opengl-driver/lib)
   '';
 
-  meta = with lib; {
+  meta = {
     description = "GTK frontend for gpu-screen-recorder.";
     mainProgram = "gpu-screen-recorder-gtk";
     homepage = "https://git.dec05eba.com/gpu-screen-recorder-gtk/about/";
-    license = licenses.gpl3Only;
-    maintainers = with maintainers; [ babbaj ];
+    license = lib.licenses.gpl3Only;
+    maintainers = [ lib.maintainers.babbaj ];
```

```suggestion
    maintainers = with lib.maintainers; [ babbaj ];
```
This is a more clean approach for future addition of maintainers.

---

Regarding this, I also like that syntax more that what I did, but there are issues and conversations to minimize use of `with`. For example this: #292468 .
Since there is only one maintaner for now, as I understand it's better not to use it, am I missing something?

---

Using `with` in a small scope like this is acceptable, just don't use it at the top level of a module where it can conflict.